spec/features/users_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
+
+feature "user can navigate the app" do
+  scenario "user can create a new account" do
+    visit new_user_path
+
+    fill_in "user_name", with: "Mary"
+    fill_in "user_email", with: "Mary@example.com"
+    fill_in "user_password", with: "secret"
+    click_on I18n.t("users.new.submit")
+
+    expect(page).to have_text(I18n.t("users.show.welcome", name: "Mary"))
+    expect(page).
+      to have_text(I18n.t(
+        "flash.actions.create.notice",
+        resource_name: "User"
+      ))
+  end
+
+  scenario "user can view their show page" do
+    user = create(:user, name: "John")
+    visit user_path(user)
+
+    expect(page).to have_text(I18n.t("users.show.welcome", name: user.name))
+  end
+
+  scenario "user can update their profile" do
```

Comment:
Nit picking: I would usually describe that as "User updates..."

Reply:
Done with sha: 5ee2ab245060a9ca7a6a25f2e6e1064039738a77